content/components/branch-name.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Branch name
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=14855%3A46652
+react: https://primer.style/react/BranchName
+description: Branch name is a label-type component rendered as an <a> tag by default that displays the name of a branch.
+
+---
+
+import {Box, Text} from '@primer/react'
+
+<img
```

Comment:
nitpick (non-blocking): I'd use the default link variant in all images/examples (except the one that shows the plain variant)

Reply:
good call, will update